tests/std/tests/P0896R4_ranges_iterator_machinery/test.cpp

```diff
@@ -11,13 +11,12 @@
 #include <utility>
 #include <vector>
 
+#include <range_algorithm_support.hpp>
+
 #define STATIC_ASSERT(...) static_assert(__VA_ARGS__, #__VA_ARGS__)
 
 namespace ranges = std::ranges;

```

Comment:
This also provides `STATIC_ASSERT` and the namespace alias:

https://github.com/microsoft/STL/blob/e7f52446c6f41fddbed31904f594a1f711f85601/tests/std/include/range_algorithm_support.hpp#L19-L21